client/state/data-layer/wpcom/jetpack/settings/test/index.js

```diff
@@ -70,14 +54,16 @@ describe( 'requestJetpackOnboardingSettings()', () => {
 		);
 	} );
 
-	test( 'should pass null token and user email in save request when site credentials are unknown', () => {
-		const getState = () => ( {
-			jetpackOnboarding: {
-				credentials: {},
+	test( 'should dispatch an action for GET HTTP request with a query including token and jpUser', () => {
```

Comment:
Let's specify what these `token` and `jpUser` are, either by prepending them in the test name with `onboarding`, or just change them to `onboarding credentials` or `onboarding auth credentials`.

Reply:
Yep, excellent call. 7d7c8d8